.jenkins/pipelines/Agnostic/Linux/Jenkinsfile

```diff
@@ -38,6 +38,37 @@ def simulationContainerTest(String version, String build_type, List extra_cmake_
     }
 }
 
+def buildCrossPlatform(String version) {
+    stage("Ubuntu ${version} OP-TEE Build") {
+        node(AGENTS_LABELS["ubuntu-nonsgx"]) {
+            timeout(GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+                def os = version == "20.04" ? "focal" : "bionic"
+                def task = """
+                           cd ${WORKSPACE}
+
+                           sdk_path=`pwd`
+                           export OE_SDK_PATH=\${sdk_path}
+                           export BUILD_PATH=\${sdk_path}/build
+                           export OPTEE_BUILD_PATH=\${sdk_path}/build/optee
+                           export PACK_PATH=\${sdk_path}/path
+                           export OS_CODENAME=${os}
+
+                           sudo ansible-playbook scripts/ansible/oe-contributors-setup-cross-arm.yml
+                           sudo apt install python python3-pyelftools p7zip-full -y
+
+                           bash devex/cross-nuget/standalone-builds/linux/build-optee.sh
+                           bash devex/cross-nuget/standalone-builds/linux/runner.sh
+                           """
+                withEnv(["OE_SIMULATION=1"]) {
```

Comment:
Since all of these variables (OE_SIMULATION, OE_SDK_PATH, BUILD_PATH, etc) are needed within the container environment, it might be cleaner to do it this way:

```
withEnv([
    "OE_SIMULATION=1",
    "OE_SDK_PATH=${WORKSPACE}",
    "BUILD_PATH=${WORKSPACE}/build",
    . . .
]) {
```